lib/membrane_mp4/movie_box/track_box.ex

```diff
@@ -274,6 +275,31 @@ defmodule Membrane.MP4.MovieBox.TrackBox do
     ]
   end
 
+  defp sample_description(%{content: %Opus{} = opus}) do
+    [
+      Opus: %{
+        children: %{
+          dOps: %{
+            fields: %{
+              version: 0,
+              output_channel_count: opus.channels,
+              pre_skip: 413,
```

Comment:
Where is this number coming from?

Reply:
Short anwser: borrowed from FFmpeg. Longer answer would be that setting the pre_skip is OPUS recomendation to set it to 80 millisececonds worth of PCM Samples at 48 kHz, so I borrowed the value from FFmpeg

Reply:
I was just curious, because [here](https://opus-codec.org/docs/opus_in_isobmff.html) I've found `312` with the same explanation about 80ms 🤔 But fair enough 😄